docs/runbooks/VirtControllerDown.md

```diff
@@ -1,40 +1,49 @@
+<!-- Edited by Jiří Herrmann, 9 Nov 2022 -->
+
 # VirtControllerDown
 
 ## Meaning
-No running virt-controller was detected for the last 5 min. Virt-controller deployment has a default replica of 2 pods.
+No running `virt-controller` pod has been detected for 5 minutes.
 
 ## Impact
-Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+Any actions related to virtual machine (VM) lifecycle management fail. This notably includes launching a new VM instance (VMI) or shutting down an existing VMI.
```

Comment:
" new VM instance (VMI) " > "new virtual machine instance (VMI)"

OCP docs will want this spelled out, even though "VM" has been spelled out in the previous sentence.